### ISSUE TITLE
document submission submission type for the existing documents to be updated when document submission feature is released

### ISSUE THREAD

**Issue by Ramu-kandimalla** [pucardotorg/dristi#2707]

**Comment by radheshjoshi1** [comment]
[evidence-script.zip](https://github.com/user-attachments/files/18225984/evidence-script.zip)
@maruthiPhanendra  provided the script for updating the submission type of evidence

**Comment by Ramu-kandimalla** [comment]
HI @manimaarans  @rajeshcherukumalli  is this script tested and verified @suresh12 @mehulseg24

**Comment by radheshjoshi1** [comment]
[evidence-script.zip](https://github.com/user-attachments/files/18254071/evidence-script.zip)
it is the updated script containing the csv file for prod also
seems like there was no evidence data for the given scenario @suresh12 
cc: @manimaarans

**Comment by utpalBeehyv** [comment]
hi @manimaarans, @radheshjoshi1 

We've made changes so that each evidence now includes the application number. Previously, the evidence was created first, followed by the application, but now we create the application first and then the evidence.

However, I have a concern: Let's say we are filing an application with five documents included. The application is created first, and then we start creating the documents as evidence. After creating two documents, the API fails while processing the remaining three. In this scenario, what do we do about the three remaining documents? how will we handle to create evidence for remaining three?

Previously, the process was designed so that if any evidence API call failed, the creation of the application wouldn't proceed. But now, with the new flow, how should we handle this situation where part of the evidence creation fails after the application has been created?

cc : @anirudh-0, @Ramu-kandimalla

**Comment by atulgupta2024** [comment]
@manimaarans i agree. now would be a good time to think BFF that manages the work between application and evidence and for the scenario of failure, can make a retry
